src/WebComponent.js

```diff
@@ -133,22 +133,12 @@ const propTypes = {
   /**
    * Enable analytics tracking.
    */
-  enableTracking: PropTypes.bool,
+  enableTracking: PropTypes.string,
```

Comment:
String because of web component?

Reply:
yes when you specify an attribute in the webcomponent it has to be string. A property can be somthing else but not an attribute.